### PR TITLE
Add guide for Org level secrets - CORE-1468

### PR DESCRIPTION
**Pull request by arsci**
Adding organization level secrets tab to Machine Users guide

### CODE REVIEW

File: _docs-sources/pipelines/security/machine-users.md [hunk 1]
```diff
@@ -82,7 +85,64 @@ The expiration of this token is up to you and the security posture of your organ
...
+The recommended path for storing secrets in GitHub Actions is with Organization level secrets, however not all GitHub plans support repository specific Organization secrets. If this is the case for your GitHub organization, you should use Repository level secrets.
+
+:::
+
+<Tabs groupId="github-actions-secrets">
+<TabItem value="Organization Secrets" label="Organization Secrets" default>
+
+1. Navigate to your top level GitHub Organization and select the **Settings** tab.
+
+1. From the navigation bar on the left side, select **Secrets and variables** then select **Actions**
```
Comment (ebeneliason) [inline _docs-sources/pipelines/security/machine-users.md:101]:
```suggestion
1. From the navigation bar on the left side, select **Secrets and variables** then select **Actions**.
```

File: _docs-sources/pipelines/security/machine-users.md [hunk 2]
```diff
@@ -82,7 +85,64 @@ The expiration of this token is up to you and the security posture of your organ
...
+  After the bootstrap process is complete, you should delete the following tokens for security purposes:
+
+    - The `PIPELINES_BOOTSTRAP` Personal Access Token from the `ci-user` GitHub account and
+    - The `PIPELINES_BOOTSTRAP_TOKEN` GitHub Actions secret from the `infrastructure-pipelines` repository
+
+  :::
+
+  :::info
+
+  For additional information on creating and using Github Actions Organization secrets, please refer to the [GitHub Documentation](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-an-organization)
```
Comment (ebeneliason) [inline _docs-sources/pipelines/security/machine-users.md:140]:
```suggestion
  For additional information on creating and using Github Actions Organization secrets, please refer to the [GitHub Documentation](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-an-organization).
```

File: _docs-sources/pipelines/security/machine-users.md [hunk 3]
```diff
@@ -82,7 +85,64 @@ The expiration of this token is up to you and the security posture of your organ
...
+
+    1. This should be assigned the _`PIPELINES_BOOTSTRAP`_ token generated by the `ci-user` in the [secrets section](#ci-user) as its value.
+
+    1. **Repository access**. Using the `Selected repositories` option select **only** the `infrastructure-pipelines` repository.
+
+  :::warning
+
+  After the bootstrap process is complete, you should delete the following tokens for security purposes:
+
+    - The `PIPELINES_BOOTSTRAP` Personal Access Token from the `ci-user` GitHub account and
```
Comment (ebeneliason) [inline _docs-sources/pipelines/security/machine-users.md:133]:
```suggestion
    - The `PIPELINES_BOOTSTRAP` Personal Access Token from the `ci-user` GitHub account
```

File: _docs-sources/pipelines/security/machine-users.md [hunk 4]
```diff
@@ -100,10 +160,20 @@ In the `infrastructure-pipelines` repository create the following secrets:
...
 
-    :::warning
-    For security purposes, you should delete:
-    1. The `PIPELINES_BOOTSTRAP` Personal Access Token from the `ci-user` GitHub account and
-    1. The `PIPELINES_BOOTSTRAP_TOKEN` GitHub Actions secret from the `infrastructure-pipelines` repository
+  :::warning
+
+  After the bootstrap process is complete, you should delete the following tokens for security purposes:
+
+    - The `PIPELINES_BOOTSTRAP` Personal Access Token from the `ci-user` GitHub account and
```
Comment (ebeneliason) [inline _docs-sources/pipelines/security/machine-users.md:167]:
```suggestion
    - The `PIPELINES_BOOTSTRAP` Personal Access Token from the `ci-user` GitHub account
```

File: _docs-sources/pipelines/security/machine-users.md [hunk 5]
```diff
@@ -82,7 +85,64 @@ The expiration of this token is up to you and the security posture of your organ
 ## Invite both machine users to Gruntwork
 Ensure both of these machine users are members of your team in **Gruntwork**’s GitHub Organization (See [instructions on inviting a user to your team](https://docs.gruntwork.io/developer-portal/invite-team#inviting-team-members) and [linking the user’s GitHub ID to Gruntwork](https://docs.gruntwork.io/developer-portal/link-github-id))
 
-## Repository Secrets
+### Configure secrets for GitHub Actions
```
Comment (ebeneliason) [inline _docs-sources/pipelines/security/machine-users.md:88]: One more thing we might want to mention in these docs is that the repository access for these org level secrets will need to be updated any time an additional live or pipelines repo is created within their org. (Likewise, repo level secrets would need to be updated in that event as well). Perhaps we can call that out at the top of this section so it's clear that this isn't a one-time thing as more repos are created?

**Review by ebeneliason — COMMENTED** [review]
Looks great overall! Left a few comments.

File: _docs-sources/pipelines/security/machine-users.md (hunk 5 above)
Comment (arsci) [reply _docs-sources/pipelines/security/machine-users.md:88]: Sounds good, I've added an additional `info` callout that mentions this

**Review by ebeneliason — APPROVED** [review]
LGTM

**Pull request merged**